cmd/openshift-install/create.go

```diff
@@ -93,7 +95,13 @@ var (
 				if err != nil {
 					return err
 				}
-				return logComplete(rootOpts.dir)
+				logrus.Info("Waiting for openshift-console route to be created...")
```

Comment:
Can we push this down into `waitForConsole`?

Reply:
(removed it, since there is a similar Infof already there in waitForConsole)